src/sentry/static/sentry/app/views/organizationGroupDetails/groupDetails.tsx

```diff
@@ -127,27 +129,11 @@ class GroupDetails extends React.Component<Props, State> {
         environments,
         projectId
       );
-      this.setState({
-        event,
-        loading: false,
-        loadingEvent: false,
-        error: false,
-        errorType: null,
-      });
+      this.setState({event, loading: false, eventError: false, loadingEvent: false});
     } catch (err) {
       // This is an expected error, capture to Sentry so that it is not considered as an unhandled error
       Sentry.captureException(err);
-      let errorType: Error = null;
-      switch (err?.status) {
-        case 404:
-          errorType = ERROR_TYPES.GROUP_NOT_FOUND;
-          break;
-        case 403:
-          errorType = ERROR_TYPES.MISSING_MEMBERSHIP;
-          break;
-        default:
-      }
-      this.setState({error: true, errorType, loading: false});
```

Comment:
Do we not want to keep this?

Reply:
errorType is used when the group endpoint fails, i had added it here, but it isn't correct. The event error is a different component that doesn't currently care about 404s.